### PR TITLE
[enocean] Adding support for Eltako TF-FKB battery status telegram

### LINKED ISSUE #4578: [enocean] Add support for Eltako TF-FKB battery status telegram
The window contact TF-FKB sends, next to the sensor telegrams, every hour a status telegram. In this telegram, the battery voltage is given, which I want to use for maintenance.

The problem is that the EEP is not given, only the content of the telegram. The nearest EEP I could find is A5-14-01, so I would call the new EEP A5_14_01_ELTAKO. Here the content like given in the manual:
ORG = 0x07
DB0 = 0x08
DB1 = 0x00
DB2 = 0x00..0xFF = 0..5V (Battery voltage)
DB3 = 0x00..0xFF = 0..5V (Energy storage)

Sample data received:

```
A5 8E A2 00 08 0580E268 0001FFFFFFFF5800
A5 8F A2 00 08 0580E268 0001FFFFFFFF5900
```
I don't know what the difference is between DB2 and DB3, so I would publish both values, adding a channel "energyStorage".

See https://github.com/fruggy83/openocean/issues/32

### PR DESCRIPTION
**Pull request by dominikkv**
Adds battery status telegram support for Eltako TF-FKB

Signed-off-by: Dominik Krickl-Vorreiter dominikkv@gmx.de
closes #4578
See https://github.com/fruggy83/openocean/pull/45

### CODE REVIEW

File: addons/binding/org.openhab.binding.enocean/ESH-INF/thing/channels.xml [hunk 1]
```diff
@@ -154,6 +154,13 @@
         <description>Voltage of the battery</description>
         <state pattern="%f %unit%" readOnly="true" />
     </channel-type>
+	
+    <channel-type id="energyStorage">
+        <item-type>Number:ElectricPotential</item-type>
+        <label>Energy Storage</label>
+        <description>?</description>
+        <state pattern="%f %unit%" readOnly="true" />
+    </channel-type>
```
Comment (wborn) [inline addons/binding/org.openhab.binding.enocean/ESH-INF/thing/channels.xml]: Please use tabs for indentation in these XML files.

File: addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_14/A5_14_01.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-2019 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (wborn) [inline addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_14/A5_14_01.java]: Please use the new EPL-v2.0 header. See also this [community topic](https://community.openhab.org/t/tutorial-update-license-to-epl-v2-0-of-your-binding-in-development-or-pull-request/65088?u=wborn).

File: addons/binding/org.openhab.binding.enocean/ESH-INF/thing/channels.xml (hunk 1 above)
Comment (dominikkv) [reply addons/binding/org.openhab.binding.enocean/ESH-INF/thing/channels.xml]: Fixed, thanks!

File: addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_14/A5_14_01.java (hunk 2 above)
Comment (dominikkv) [reply addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_14/A5_14_01.java]: Added the header for both new files. Thanks!

**Comment by dominikkv** [comment]
Thank you, @wborn, for your feedback. I think I have fixed all you have suggestioned, furthermore I did a rebase because of some conflicts that arose lately. If I can do anything more please don't hesitate to ask.

**Pull request merged**